### DISCUSSION TITLE
Browser not supported in safari

### DISCUSSION THREAD

**Discussion by Kundannetset** [deepgram/community#615] [General help]
### Which Deepgram product are you using?

Deepgram SDKs

### Details

Browser not supported in safari
![Screenshot 2024-02-20 at 5 47 05 PM](https://github.com/deepgram/community/assets/145329403/2e4c1810-352b-4e1c-9c96-9b020614f9ee)

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@kundannetsetDemo  can you give us a bit more context into what you are building and which SDK you are trying to use? The more details the better for us to assist.
